cppcache/include/geode/CacheableBuiltins.hpp

```diff
@@ -528,34 +346,63 @@ class _GEODE_EXPORT CacheableArray : public Cacheable {
         apache::geode::client::serializer::objectArraySize(m_value));
   }
 
- private:
-  CacheableArray(const CacheableArray& other) = delete;
-  CacheableArray& operator=(const CacheableArray& other) = delete;
+private:
+
+  _GEODE_FRIEND_STD_SHARED_PTR(CacheableArray)
+
   std::vector<T> m_value;
 
- public:
-  inline CacheableArray() {}
-  inline CacheableArray(int32_t length) : m_value(length) {}
-  inline CacheableArray(std::vector<T> value) : m_value(value) {}
+public:
+
+  inline const std::vector<T>& value() const {
+    return m_value;
+  }
+
+  inline int32_t length() const {
+    return m_value.size();
+  }
 
-  inline const std::vector<T> value() const { return m_value; }
-  inline int32_t length() const { return m_value.size(); }
   static std::shared_ptr<Serializable> createDeserializable() {
     return std::make_shared<CacheableArray<T, GeodeTypeId>>();
   }
-  inline static std::shared_ptr<CacheableArray<T, GeodeTypeId>> create() {
+
+  static std::shared_ptr<CacheableArray<T, GeodeTypeId>> create() {
```

Comment:
Any reason for dropping the inline?

Reply:
No reason really.  I just didn't think explicit `inline` added much value here.